lib/types.py

```diff
@@ -0,0 +1,22 @@
+from typing import List, Iterable
+
+try:
+    from typing import TypeAlias  # type: ignore
+except ImportError:
+    # for Python < 3.10
+    from typing_extensions import TypeAlias  # type: ignore
+
+import shapely
+
+"""The datatype representing a connected 2D shape in the plane."""
+Shape = shapely.geometry.Polygon
```

Comment:
I wonder if we shouldn't name this differently, because we have non polygon shapes for the other stitch types.
So this maybe this is a PolygonShape or FillShape or something like that.